sources/academy/webscraping/scraping_basics_python/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Web scraping basics for Python devs
+description: Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.
+sidebar_position: 10
+category: web scraping & automation
+slug: /scraping-basics-python
+---
+
+import DocCardList from '@theme/DocCardList';
+
+**Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.**
+
+---
+
+:::danger Work in progress
+
+This course is incomplete. As we work on adding new lessons, we would love to hear your feedback. Comment right here under each page or [file a GitHub Issue](https://github.com/apify/apify-docs/issues) to discuss a problem.
+
+:::
+
+In this course we'll use Python to create an application for watching prices. It'll be able to scrape all product pages of an e-commerce website and record prices. Data from several runs of such program would be useful for seeing trends in price changes, detecting discounts, etc.
+
+<!--
+TODO image of warehouse with some CVS or JSON exported, similar to sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-collection.png, which is for some reason the same as sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-extraction.png
+-->
+
+![E-commerce listing on the left, JSON with data on the right](./images/scraping.png)
+
+## What you'll do
+
+- Inspect pages using browser DevTools
+- Download web pages using the HTTPX library
+- Extract data from web pages using the Beautiful Soup library
+- Save extracted data in various formats, e.g. CSV which MS Excel or Google Sheets can open
+- Follow links programatically (crawling)
+- Save time and effort with frameworks, such as Scrapy, and scraping platforms, such as Apify
```

Comment:
I wrote this sentence before Crawlee for Python has existed 😀 We talked about it with @souravjain540 at EuroPython and I tend to agree we should choose Crawlee, even though it's very new, because it's just easier to grasp.

Ideally, Crawlee for Python docs would feature pages like "Crawlee vs Scrapy" or "Migrating from Scrapy", which would explain how to write the same thing in both, or guide people like me who are fluent in Scrapy and want to try or migrate to Crawlee. First idea was it could be a separate course in the academy, but after giving it a thought I'd say its best home is the Crawlee docs.

Reply:
I wouldn't use both in the course for beginners. First I think the code could be so different than even using tabs could result in a mess, second I strongly believe beginners diving to a new topic need to be shown a path without crossroads and options, and the author of the course is responsible for making all the hard choices for them.

Reply:
Addressed in https://github.com/apify/apify-docs/pull/1023/commits/307121a1a501105a6a323e63ffdc559caa42a628